src/pudl/metadata/resources/ferc1.py

```diff
@@ -437,6 +437,29 @@
         "etl_group": "ferc1",
         "field_namespace": "ferc1",
     },
+    "depreciation_amortization_summary_ferc1": {
+        "description": "Summary of depreciation, and amortization of electric plant.",
+        "schema": {
+            "fields": [
+                "utility_id_ferc1",
+                "report_year",
+                "functional_classification",
+                "depreciation_expense",
+                "depreciation_expense_asset_retirement",
+                "amortization_limited_term_electric_plant",
+                "amortization_other_electric_plant",
+                "depreciation_amortization_total",
```

Comment:
If only the DBF has the total column, do we want to ensure that we get the same data across both datasets either by dropping the total from the outputs (after verifying that the DBF data all actually adds up correctly...) or by producing a calculated total column for the XBRL data?

Reply:
Yeah I was thinking we might want to do that. Do you have any preference between computing the total for XBRL vs dropping DBF totals?

Reply:
@cmgosnell what do you think we should do here? Have totals only from the DBF? Calculate totals for the XBRL? Drop totals from the DBF?

Reply:
I chatted w/ Christina. Let's just leave it as it's reported for the moment, with a total in the DBF data, and no total in the XBRL.